pkg/master/import_known_versions.go

```diff
@@ -38,3 +40,8 @@ import (
 	_ "k8s.io/kubernetes/pkg/apis/settings/install"
 	_ "k8s.io/kubernetes/pkg/apis/storage/install"
 )
+
+// apis internal to the apiserver must be explicitly passed the scheme
+func init() {
+	auditinstall.Install(legacyscheme.Scheme)
```

Comment:
cc @deads2k @sttts for use of legacyscheme in init

Reply:
odd one-off. Why is this different?

Reply:
So, I originally followed the same pattern but hit an issue with a verification test the doesn't allow the apiserver to import from `k/pkg`. Another option would be to possibly hack around that test

Reply:
I understand you motivation. Do we need to install into legacyscheme at all? It's legacy for a reason: we should not use it anymore.

Reply:
@sttts I've tried to create a new scheme for the apiserver types, and hit a number of tricky problems. It seems that breaking from legacy scheme is will require quite a bit of rework. I'm wondering if you would be ok with splitting that out into a separate PR and inheriting legacy scheme for the interim?

Reply:
So first issue was applying the `runtime-config` options https://github.com/kubernetes/kubernetes/blob/master/cmd/kube-apiserver/app/server.go#L457, I found a way around this that involves taking multiple schemes and merging them but I'm not sure its the best solution. Next, is the issue of the storage config https://github.com/kubernetes/kubernetes/blob/master/cmd/kube-apiserver/app/server.go#L494 that bakes in legacy scheme today. It seems like I may need a fully separate config for the api server similar to the [apiextenstions server] (https://github.com/kubernetes/kubernetes/blob/master/cmd/kube-apiserver/app/apiextensions.go#L33), and add it to the chain. Please advise if I'm off base here

Reply:
ok tried both directions here, but option 2 looks the cleanest. PTAL